tuxemon/menu/interface.py

```diff
@@ -141,6 +141,48 @@ def in_focus(self, value: bool) -> None:
         self._in_focus = bool(value)
 
 
+class MenuTechnique(Generic[T], Sprite):
```

Comment:
Is there no way that you can do this without duplicating the contents of another class?  last i knew, there was some work being done to remove the menus and replace them with another lib.  with that in mind, we should avoid adding obsolete code

Reply:
I'm going to remove it and stick to MenuItem without adding obsolete code.

Reply:
You were right, much better, less files involved.